osmaxx/geodesy/coordinate_reference_system.py

```diff
@@ -0,0 +1,99 @@
+from django.contrib.gis.geos.collections import MultiPolygon
+from django.contrib.gis.geos.polygon import Polygon
+
+from osmaxx.conversion_api.coordinate_reference_systems import WGS_84
+
+MIN_LONGITUDE_DEGREES = -180
+MAX_LONGITUDE_DEGREES = +180
+
+
+class UniversalTransverseMercatorZone:
+    HEMISPHERE_PREFIXES = dict(
+        north=326,
+        south=327,
+    )
+    NUMBER_OF_ZONES_PER_HEMISPHERE = 60
+    VALID_ZONE_NUMBERS = range(1, NUMBER_OF_ZONES_PER_HEMISPHERE + 1)
+    ZONE_WIDTH_DEGREES = 360 / NUMBER_OF_ZONES_PER_HEMISPHERE
+    MAX_LONGITUDE_OFFSET = 90.0 - 9.9e-14
```

Comment:
We have two GEOS/Proj.4 implementations/versions involved: The one GeoDjango uses and the one PostGIS uses.

In my tests, PostGIS was able to transform WGS-84 coordinates to UTM iff the longitude was within up to 90° of the chosen UTM zone's "central meridian". However, with GeoDjango, it worked only _nearly_ up to 90°, at least on my machine. The difference from 90° (9.9&times;10<sup>-14</sup>°) was found by trial and error. Are more precise value could probably be found by systematic (e.g. automated) bisection, but I deemed the value I found close enough for our purposes.